### PR TITLE
Release 2.7.15

### PR DESCRIPTION
**Pull request by jakirkham**
https://www.python.org/downloads/release/python-2715/

### CODE REVIEW

**Comment by jakirkham** [comment]
Hmm...did some Python 3 code sneak into the Python 2 codebase on Windows. ;)

```python
C:\bld\python_1525666272216\work\Lib\xml\sax\__init__.py
1994 File(s) copied
        1 file(s) moved.
        1 dir(s) moved.
        1 file(s) moved.
        1 dir(s) moved.
Compiling C:\bld\python_1525666272216\_h_env\Lib\tkinter\ttk.py ...
  File "C:\bld\python_1525666272216\_h_env\Lib\tkinter\ttk.py", line 83
    for *state, val in items:
        ^
SyntaxError: invalid syntax
```

**Comment by jjhelmus** [comment]
> did some Python 3 code sneak into the Python 2 codebase on Windows

This file was included in python/cpython#6326 to close [bpo 33906](https://bugs.python.org/issue33096).  I haven't had time to dig deep enough to know if this new file is required for the fix or not.  

For `defaults` we skipped byte compiling this file by [adding it to the compileall.py regex](https://github.com/AnacondaRecipes/python-feedstock/pull/10/commits/8bdacc7519d81fe610621012678939c1e21e6782).

**Comment by jakirkham** [comment]
Thanks @jjhelmus.

Have added [a comment upstream]( https://github.com/python/cpython/pull/6326#issuecomment-387253019 ). Let's see what they say.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -15,8 +15,10 @@ source:
       - win-library_bin.patch           # [win]
       - 0008-Win32-Fixes-for-Windows-GCC-interop-needed-by-RPy2-a.patch
       - 0017-Fix-find_library-so-that-it-looks-in-sys.prefix-lib-.patch
+      - PR_6724.patch
```
Comment (jakirkham) [inline recipe/meta.yaml]: Generated from PR ( https://github.com/python/cpython/pull/6724 )'s only commit ( https://github.com/python/cpython/commit/6af2231df973dd2fa43d29528086d20eb90e35b5 ).

**Comment by jjhelmus** [comment]
LGTM, thanks for the update @jakirkham

**Comment by jakirkham** [comment]
Thanks @jjhelmus :)

**Pull request merged**